glide.lock

```diff
@@ -167,7 +160,7 @@ imports:
 - name: github.com/Sirupsen/logrus
   version: 55eb11d21d2a31a3cc93838241d04800f52e823d
 - name: github.com/spf13/pflag
-  version: 8e7dc108ab3a1ab6ce6d922bbaff5657b88e8e49
+  version: 5644820622454e71517561946e3d94b9f9db6842
```

Comment:
i think i tried doing that and had some other issue because of which i had to change the glide.lock file....may be we can try again to pin in the glide.yaml and see